src/main/java/io/github/sefiraat/networks/slimefun/tools/NetworkRake.java

```diff
@@ -1,13 +1,7 @@
 package io.github.sefiraat.networks.slimefun.tools;
 
 import io.github.sefiraat.networks.Networks;
-import io.github.sefiraat.networks.slimefun.network.NetworkBridge;
-import io.github.sefiraat.networks.slimefun.network.NetworkExport;
-import io.github.sefiraat.networks.slimefun.network.NetworkGrabber;
-import io.github.sefiraat.networks.slimefun.network.NetworkImport;
-import io.github.sefiraat.networks.slimefun.network.NetworkMonitor;
-import io.github.sefiraat.networks.slimefun.network.NetworkObject;
-import io.github.sefiraat.networks.slimefun.network.NetworkPusher;
+import io.github.sefiraat.networks.slimefun.network.*;
```

Comment:
No star imports please. This may well be automatic from your IDE which you will need to turn off before importing the other required classes, otherwise good shout adding these in :)

Reply:
Uh, oh. Let me fix it